docs/Tutorial.md

```diff
@@ -722,9 +756,7 @@ Let's get back to the post list for a minute. It offers sorting and pagination,
 React-admin can use Input components to create a multi-criteria search engine in the list view. Pass an array of such Input components to the List `filters` prop to enable filtering:
 
 ```jsx
-// in src/posts.js
-import { ReferenceInput, TextInput, List } from 'react-admin';
```

Comment:
why remove this import?

Reply:
Because the import doesn't change from the previous step, and it doesn't look like that in the previous step. As is, the new import is misleading.